corehq/apps/export/views.py

```diff
@@ -488,8 +488,11 @@ def download_export_form(self):
         raise NotImplementedError("You must implement download_export_form.")
 
     @staticmethod
-    def get_export_schema(export_id):
-        return get_document_or_404_lite(SavedExportSchema, export_id)
+    def get_export_schema(domain, export_id):
+        doc = get_document_or_404_lite(SavedExportSchema, export_id)
```

Comment:
you can use this instead if you have the domain: https://github.com/dimagi/commcare-hq/blob/master/corehq/util/couch.py#L51